<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewMember</class>
 <widget class="QDialog" name="AddNewMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>456</width>
     <height>306</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="memberIDLabel">
      <property name="text">
       <string>Member ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="memberIDLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="memberNameLabel">
      <property name="text">
       <string>Member Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="memberNameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="memberSurnameLabel">
      <property name="text">
       <string>Member Surname</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="memberSurnameLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Member Sex</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_male">
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_female">
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dateOfBirthLabel">
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="membershipDurationLabel">
      <property name="text">
       <string>Membership Duration (Month)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
